src/discovery.jl

```diff
@@ -253,4 +253,5 @@ function show_next_transition(io::IO, tz::TimeZone=localzone())
     show_next_transition(io, @mock now(tz))
 end
 
+show_next_transition(io::IO, x...) = throw(MethodError(show_next_transition, (io, x...)))
```

Comment:
Why haven't you just added a type on `x` below?

Reply:
Also worked. I was trying to be generic if I added additional methods of the form `show_next_transition(io::IO, x)`

Reply:
I guess I could also do that: `show_next_transition(x) = show_next_transition(stdout, x)`